server/index.js

```diff
@@ -1,11 +1,12 @@
-var express = require('express');
-var app = express();
-var port = 3000;
-var bodyParser = require('body-parser');
-var path = require('path');
-var SingleOwner = require('../database/schema.js').SingleOwner;
+const express = require('express');
 
-app.use(bodyParser.urlencoded({extended: false}));
+const app = express();
+const port = 3117;
+const bodyParser = require('body-parser');
```

Comment:
@SimonC989 @carl12 In the same way bodyParser is one of those default middleware to add to a new express server, it will likely be useful to get into the habit of adding logging middleware (like morgan) to every new server.